tunnistamo/settings.py

```diff
@@ -29,6 +29,11 @@
     MEDIA_URL=(str, '/media/'),
     NODE_MODULES_ROOT=(str, os.path.join(BASE_DIR, 'node_modules')),
 
+    COOKIE_SECURE=(bool, True),
+    COOKIE_PREFIX=(str, 'tunnistamo'),
```

Comment:
Should this be named `COOKIE_NAME_PREFIX`?

Reply:
Done